runtime/compiler/runtime/Runtime.cpp

```diff
@@ -623,8 +623,8 @@ JIT_HELPER(_nativeStaticHelper);
 JIT_HELPER(jitLookupInterfaceMethod);
 JIT_HELPER(jitMethodIsNative);
 JIT_HELPER(jitMethodIsSync);
-JIT_HELPER(jitPreJNICallOffloadCheck);
-JIT_HELPER(jitPostJNICallOffloadCheck);
+JIT_HELPER(fast_jitPreJNICallOffloadCheck);
+JIT_HELPER(fast_jitPostJNICallOffloadCheck);
```

Comment:
Do not declare these as JIT_HELPER.  They aren't assembly functions.  They should be externed as proper C functions.